akka-sample-kafka-to-sharding-scala/processor/src/main/scala/sample/sharding/kafka/UserEventsKafkaProcessor.scala

```diff
@@ -68,4 +50,38 @@ object UserEventsKafkaProcessor {
   }
 
 
+  private def consumeFromTopic(shardRegion: ActorRef[UserEvents.Command], processorSettings: ProcessorSettings)
```

Comment:
Is it perhaps "startConsumingFromTopic" given that it materializes a stream that will be running after it returns?

Reply:
done